src/dvclive/lightning.py

```diff
@@ -119,6 +123,23 @@ def log_metrics(self, metrics: Dict[str, Any], step: Optional[int] = None):
                 self.experiment._latest_studio_step -= 1  # noqa: SLF001
             self.experiment.next_step()
 
+    def after_save_checkpoint(self, checkpoint_callback: ModelCheckpoint) -> None:
+        self._checkpoint_callback = checkpoint_callback
+        if self._log_model == "all" or (
+            self._log_model is True and checkpoint_callback.save_top_k == -1
+        ):
+            self.experiment.log_artifact(checkpoint_callback.dirpath)
+
     @rank_zero_only
     def finalize(self, status: str) -> None:
+        checkpoint_callback = self._checkpoint_callback
+        # Save model checkpoints.
+        if self._log_model is True:
+            self.experiment.log_artifact(checkpoint_callback.dirpath)
+        # Log best model.
```

Comment:
WDYT of creating a copy in "dvclive" folder (or in the checkpoints folder itself), at least for the best?

It seems that we would be changing the path of the registered model between experiments in the current behavior

Reply:
Yes, which I guess is also what other trackers do AFAICT? Do you think the path matters? Maybe it makes it easier to `dvc get` later, although we could make that work by the artifact name. No strong opinion from me.